util.c

```diff
@@ -5322,3 +5286,159 @@ int util_is_wlan(char *device)
   return slist_getentry(config.net.wlan.devices, device) ? 1 : 0;
 }
 
+
+/*
+ * Write log message.
+ *
+ * level is a bitmask determining the destination to log to.
+ *
+ * Add a time stamp when the destination has the LOG_TIMESTAMP flag set.
+ */
+void util_log(unsigned level, char *format, ...)
```

Comment:
Yes, I considered. But for logging to the console I want to keep the option not to have a newline.